package.json

```diff
@@ -16,6 +16,8 @@
   "dependencies": {
     "babel-eslint": "^10.1.0",
     "babel-polyfill": "^6.26.0",
+    "babel-core": "6.26.3",
+    "babel-loader": "8.2.2",
```

Comment:
Curious what prompted adding these? Also, we probably should not be setting the version numbers at fixed values.